src/main/java/com/ericsson/ei/frontend/utils/BackEndInstanceFileUtils.java

```diff
@@ -34,121 +30,41 @@
 
     private String eiInstancesPath;
 
-    @Autowired
-    BackEndInstancesUtils backendInstancesUtils;
-
     @Value("${ei.backend.instances.filepath:#{null}}")
     private String backendInstancesFilePath;
 
-    @Value("${ei.backend.instances.list.json.content:#{null}}")
-    private String backendInstancesListJsonContent;
-
     @PostConstruct
-    public void init() throws IOException {
-        LOG.info("Initiating BackEndInstanceFileUtils.");
-
-        // Use home folder if a specific backendInstancesFilePath isn't provided
-        boolean useUserHomeDirectory = (backendInstancesFilePath == null || backendInstancesFilePath.isEmpty());
-        if (useUserHomeDirectory) {
-
-            String homeFolder = System.getProperty("user.home");
-            String eiHome = Paths.get(homeFolder, EI_HOME_DEFAULT_NAME).toString();
+    public void backEndInstanceFileUtils() {
+        try {
+            LOG.info("Initiating BackEndInstanceFileUtils.");
 
-            Boolean eiHomeExists = Files.isDirectory(Paths.get(eiHome));
-            if (!eiHomeExists) {
-                createEiHomeFolder(eiHome);
-            }
+            // Use home folder if a specific backendInstancesFilePath isn't provided
+            boolean useUserHomeDirectory = (backendInstancesFilePath == null || backendInstancesFilePath.isEmpty());
```

Comment:
this variables can be final as they are not modified

Reply:
check in the other places also where we don't modify the variables.

Reply:
Oh ye, will do 👍